### PR TITLE
[3party] Update protobuf to v4.23.4

### PR DESCRIPTION
**Pull request by AndrewShkrob**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Kothic Mapcss parser/processor tailored for Organic Maps use.
+
+Install dependencies:
```
Comment (pastk) [inline README.md]: Please mention it requires Python >= 3.8

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+protobuf==4.23.4
```
Comment (pastk) [inline requirements.txt]:
```suggestion
protobuf~=4.23.0
```

We're going to have at least minor updates, right?

**Review by pastk — COMMENTED** [review]
The added/removed zeros issue should be solved.
LGTM otherwise!

**Review by pastk — APPROVED** [review]

**Comment by AndrewShkrob** [comment]
@biodranik @vng merge?

**Review by vng — APPROVED** [review]

**Pull request merged**